### ISSUE TITLE
Error with line-length rule

### ISSUE THREAD

**Issue by PabloAlexis611** [adrienverge/yamllint#522]
Hello there! I'm new to `yamllint`. I've read the documentation in regards to the `line-length` rule.
Funny enough, I was adding some comments to my own `.yamllint` in regards to the `line-length` rule itself.

This was my `.yamllint` configuration:

``` yaml
---
# Extend from default yamllint configuration.
# See: https://yamllint.readthedocs.io/en/stable/configuration.html#default-configuration
extends: default

# Customized yamllint rules.
rules:
  # See: https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.line_length
  line-length:
    allow-non-breakable-words: true
    allow-non-breakable-inline-mappings: true
```

I originally expected this to pass. But line #_3_ and #_8_ throw the error that lines are too long (since I'm using the default for that).
Looking back to the examples [provided here](https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.line_length), I notice that for the `allow-non-breakable-words` and `allow-non-breakable-inline-mappings` examples, the documentation has an example like this instead:

```
# https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.line_length
```

So, I fixed the error by removing the "See: " portion in my YAML comments.
But my question is, is this intended? Shouldn't non-breakable stuff like URLs there be ignored in the `line-length` context when having those rules set?

If not, then maybe a simple documentation update with an example that _fails_ would be helpful

**Comment by adrienverge** [comment]
Thanks @andrewimeson, I couldn't say it better :)
+1 if you want to make the documentation clearer.

**Comment by adrienverge** [comment]
In the meantime I'll close this issue, since this is the expected behavior.
